### PR TITLE
Fix sentences

### PR DESCRIPTION
**Pull request by AlexDzot**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: russian/sleep/brushing/choice/blue-toothbrush.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+После того, как вы взяли в руку синюю щетку, комната обращается вверх тормашками и вы начинаете увязать в зефире.
```
Comment (casey-collab) [inline russian/sleep/brushing/choice/blue-toothbrush.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `russian/sleep/brushing/choice/red-toothbrush.md` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**